packages/vscode-extension/package.json

```diff
@@ -1547,6 +1547,11 @@
           ],
           "default": "Info",
           "markdownDescription": "Set the log level of Teams Toolkit. The default value is `Info`. The available values are `Debug`, `Verbose`, `Info`. The definitions of the log levels are:\n\n`Debug`:\n\n- HTTP request and response details from Teams Toolkit to external services such as Azure, Microsoft 365, and Teams Developer Portal.\n- Information related to Microsoft 365 and Azure accounts and access permissions.\n- Debugging information for each Teams Toolkit command execution, such as stack trace, ARM deployment information, etc.\n\n`Verbose`:\n\n- Progress information when executing lifecycle commands defined in the YAML file.\n- Execution details of each action that defined in the YAML file, including outcomes, result summaries, diagnostic information, etc.\n\n`Info`: Teams Toolkit command execution summaries.\n"
+        },
+        "fx-extension.enableMicrosoftKiota": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "Use Microsoft Kiota to generate API Plugin from a local OpenAPI description or search for public APIs. If you don't have Microsoft Kiota installed, please install [here](https://marketplace.visualstudio.com/items?itemName=ms-graph.kiota)"
```

Comment:
Use Microsoft Kiota to generate an API Plugin from a local OpenAPI description or search for public APIs. If Microsoft Kiota isn't installed, install it from the [Visual Studio Marketplace] (https://marketplace.visualstudio.com/items?itemName=ms-graph.kiota)